IdentityCore/src/webview/systemWebview/ios/MSIDAuthenticationSession.m

```diff
@@ -38,7 +38,9 @@
 #import "MSIDBackgroundTaskManager.h"
 #if !MSID_EXCLUDE_WEBKIT
 #import <SafariServices/SafariServices.h>
-#import <AuthenticationServices/AuthenticationServices.h>
+    #if __IPHONE_OS_VERSION_MAX_ALLOWED >= 130000
```

Comment:
why was this added? This framework is available since iOS 12 and Xcode 10.

Reply:
right, changed.